src/main/java/com/microsoft/graph/tasks/LargeFileUploadTask.java

```diff
@@ -98,12 +100,13 @@
      * @param inputStream     the input stream
      * @param streamSize      the stream size
      * @param uploadTypeClass the upload type class
+     * @throws GraphServiceException if there is an interruption in the service
      */
     public LargeFileUploadTask(@Nonnull final IUploadSession uploadSession,
                                  @Nonnull final IBaseClient<?> client,
                                  @Nonnull final InputStream inputStream,
                                  final long streamSize,
-                                 @Nonnull final Class<UploadType> uploadTypeClass) {
+                                 @Nonnull final Class<UploadType> uploadTypeClass) throws GraphServiceException {
```

Comment:
binary breaking as well. I believe this was pushed here because of the change in handler and will be safely reverted once the change in handler is reverted